lms/resources/_js_config/__init__.py

```diff
@@ -288,14 +288,6 @@ def enable_dashboard_mode(self, token_lifetime_seconds: int) -> None:
                             "api.dashboard.assignments.grading.sync"
                         ),
                     ),
-                    auto_grading_sync_enabled=self._lti_user
```

Comment:
Note that this is "if the user is not staff and the setting is enabled".

I reckon we should remove the flag and switch the FE to read both the `is_staff` and `is_gradable` values.

Reply:
> I reckon we should remove the flag and switch the FE to check:
> 
> * User is not `is_staff`
> * Assignment uses auto grading
> * Assignment is `is_gradable` .

Done: https://github.com/hypothesis/lms/pull/6873/files#diff-7eeb51367daff96948ffa40e42cb086420b606a7f2e8f3a0ce6cd50c05d84bebR339